pgp/keysource_test.go

```diff
@@ -24,6 +24,21 @@ var (
 	mockFingerprint = "B59DAF469E8C948138901A649732075EA221A7EA"
 )
 
+func TestGnuPGHome_Import_With_Missing_Binary(t *testing.T) {
```

Comment:
Should be placed below `TestGnuPGHome_Import`.

Reply:
Also wondering what a failed import looks like now with a working GnuPG binary path, as I am not sure what gets logged to `stdout`.

Reply:
Good point. I added an assertion to the existing case `TestGnuPGHome_Import`.